changelogs/unreleased/gh-7932-recovery-mixed-transactions.md

```diff
@@ -0,0 +1,5 @@
+## feature/replication
+
+* This patch implements correct recovery of mixed transactions. To do this,
+  use `force_recovery` = true. If you wish to revert to the old behavior,
+  you must not set the `force_recovery` (gh-7932).
```

Comment:
Please also check what happens when one node feeds the other such a broken xlog.
It seems a similar mechanic should be added to applier.

Reply:
To replication do the following. Recover the broken xlog node by setting `force_recovery` to `true`. Then do replica rebootstrap.

Reply:
Please, mention that in the changelog and docbot request with a proper explanation.
That one is too short and lacks context.
If I were the user, I'd have questions. To replication from which node to which node? Which replica should be rebootstrapped? The just recovered one or some other?